example/types/ExampleContract.py

```diff
@@ -50,12 +50,13 @@
 
 from pypechain.core import combomethod_typed, dataclass_to_tuple, get_abi_input_types, rename_returned_types
 
-from .ExampleTypes import FlipEvent, FlopEvent, InnerStruct, NestedStruct, SimpleStruct
+from . import ExampleTypes as Example
```

Comment:
why use the alias here? we are the ones that name it ExampleTypes for Example.sol right?

Reply:
The alias here is to keep the full python reference the same that's defined in the ABI (e.g., `Example.MyStruct`)